scss/_title-bars.scss

```diff
@@ -0,0 +1,70 @@
+.title-bar {
+
+  // Organize the title bar
+  %container-flex-properties {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  > .container,
+  > .container-fluid {
+    @extend %container-flex-properties;
+  }
+
+  @each $breakpoint, $container-max-width in $container-max-widths {
+    > .container#{breakpoint-infix($breakpoint, $container-max-widths)} {
+      @extend %container-flex-properties;
+    }
+  }
+
+  // Title bar's objects sizes
+  img,
+  svg {
+    width: auto;
+    height: $title-bar-image-ratio;
+  }
+
+  h1,
+  & {
+    font-size: $h2-font-size;
+    line-height: $h2-line-height;
+    letter-spacing: $h2-spacing;
+
+    @include media-breakpoint-up(md) {
+      font-size: $display2-size;
+      line-height: $display-line-height;
+      letter-spacing: $display2-spacing;
+    }
+
+    @include media-breakpoint-up(xl) {
+      font-size: $display1-size;
+      letter-spacing: $display1-spacing;
+    }
+  }
+
+  // Spacing of the title
+  h1 {
```

Comment:
* It seems that there is a 1px difference (vertical alignment) with the [design guidelines](https://system.design.orange.com/0c1af118d/p/774477-local-headers/b/743cd0/i/66611057) in all modes. I think the title should be lower by 1px :thinking: → I observe the same issue with the vertical alignment of the image.
* From 480px (equal and below), it seems that the title is 1px too on the right. I think the title should be more to the left of 1px :thinking:

Reply:
The image's height is a ratio of Zeplin's image height and font-size (which is around 1.8). As this is not pixel perfect, do you think that an harcoded value would be better ?